app/tasks/batch.py

```diff
@@ -1,29 +1,30 @@
 from typing import Any, Dict, List, Optional, Callable, Awaitable, Set
 from time import sleep
 from datetime import datetime
+import os
 
 from app.models.pydantic.job import Job
 
+POLL_WAIT_TIME = 30
 BATCH_CLIENT = None
+REGION = os.environ.get("REGION", "us-east-1")
```

Comment:
I created a module app/settings/global.py where we can import all the env variables. You can then import then from there

form ..settings.globale import REGION